webapp/src/components/rhs/user_badges.tsx

```diff
@@ -0,0 +1,139 @@
+import {UserProfile} from 'mattermost-redux/types/users';
+import React from 'react';
+
+import Scrollbars from 'react-custom-scrollbars';
+
+import {BadgeID, UserBadge} from '../../types/badges';
+import Client from '../../client/api';
+
+import {RHSState} from 'types/general';
+import {RHS_STATE_DETAIL} from '../../constants';
+
+import UserBadgeRow from './user_badge_row';
+
+import './user_badges.scss';
+
+type Props = {
+    isCurrentUser: boolean;
+    user: UserProfile | null;
+    actions: {
+        setRHSView: (view: RHSState) => void;
+        setRHSBadge: (badge: BadgeID | null) => void;
+    };
+}
+
+type State = {
+    loading: boolean;
+    badges?: UserBadge[];
+}
+
+function renderView(props: any) {
```

Comment:
We are repeating this pattern a lot, would make sense to have your own Scrollbar component warpping this repetition?

Actually the rest of the parameters are the same too, so you can make it just a simple `<Scrollbars>{content}<Scrollbars>` encapsulating that logic in your own wrapper component.